packages/common/src/index.ts

```diff
@@ -101,6 +103,11 @@ export enum ConsensusAlgorithm {
   Casper = 'casper',
 }
 
+export type CliqueConfig = {
+  period: number
+  epoch: number
+}
+
```

Comment:
Wondered if this should really go here and then I re-realized that we are really mixing a lot of things up regarding type, options dicts and main class location, we should re-structure this to match other libraries like tx or block.

Will open an own dedicated issue though, not something to be done here.

Reply:
I went ahead and moved this config back to `./types.ts` and added the `EthashConfig` and `CasperConfig` empty types as well.  All the tests pass locally so  have pushed a commit.  It also necessitated re-exporting `CliqueConfig` from `index.ts` but I think that gives us a point of departure on the general `Common` reorganization issue.